### DISCUSSION TITLE
Organization Update

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [PrismarineJS/mineflayer#2980] [Ideas]
# Website

For the website I propose to use a Nextjs infrastructure so with React.
Ref: [prismarinejs.github.io #68](https://github.com/PrismarineJS/prismarinejs.github.io/pull/68)

# Docs

For our documentations I propose to have an infra with [nextra](https://nextra.site).
It allows a file structure of this type `pages/file.<lang>.<md|mdx>`

The idea is to have a more modern and flexible infrastructure.

[demo/draft of this doc with nextra](https://github.com/AugustinMauroy/beta-mineflayer-docs)

# Admin/TSC

I propose to create a repo which could be called TSC or Admin or other.

It would take care of, contain everything that concerns the governance of the project. This is where decisions would be made e.g. "do we migrate all our infra to nextjs? 

He will also welcome new contributors.

# Organization

The Organization should be "reorganized".  To allow the project to grow.

* The creation of a TSC/admin
* Better branding to apply to all websites and other
* Better governance
* We could create regular meetings with the collaborator to move forward.

# To conclude

I am listening to your opinions. I will update this text according to your opinions and areas of misunderstanding.

The purpose of this is of course to move forward.

>I really want to advance the project. Unfortunately my technical skills limit me on the help I could bring to the project like MineFlayer FlyingSquid...
>but I wish to help on the direction the project will take. And of course to help from a website, docs point of view.

**Comment by rom1504** [comment]
Website: as said before, we can change the infra *if* it helps to provide better features, so you need to implement these features, before anything get merged

Docs: try it on one repo, let's see if it makes things better

Admin: we have prismarine-contribute and prismarine-meta but I am not convinced it helps. Using mineflayer is fine for these discussions

Organisation : i am definitely against introducing meetings. We have been organizing for a long time via chat, it works great

Overall: pick an objective if you want to improve things and focus on this

What is the goal ?

**Reply by AugustinMauroy** [reply]
Website: I think merge the infra then add features would be smarter. After that it's up to the maintainer. And I also understand your point of view because there is no guarantee that I will add these features. 

Docs: I will create an demo.

Admin: I think we should unify in an admin that would contain [PrismarineJS/prismarine-design](https://github.com/PrismarineJS/prismarine-design), [PrismarineJS/prismarine-meta](https://github.com/PrismarineJS/prismarine-meta). And [PrismarineJS/prismarine-contribute](https://github.com/PrismarineJS/prismarine-contribute) should be archived because if we want to put a general document (for the organization) it should go on the admin repo and each repo has its Guideline contribution.

Organisation : Honestly I think that putting one on every now and then is just a way to meet people and talk to them.

Overall: I list but idea to advance the projects in time. After everyone has given their opinion we will move on. I think we have to go step by step.

Thank you for giving your POV
